### PR TITLE
actions: update helm actions to use release 1.0.0

### PR DESCRIPTION
**Pull request by cpanato**
- Update actions to use release 1.0.0
- update helm 2 to release 2.16.9

### CODE REVIEW

File: .github/workflows/lint-test.yaml [hunk 1]
```diff
@@ -39,16 +39,16 @@ jobs:
 
       - name: Run chart-testing (lint)
         id: lint
-        uses: helm/chart-testing-action@v1.0.0-alpha.3 # fixed on v1.0.0-alpha.3 for helm2
+        uses: helm/chart-testing-action@v1.0.0
```
Comment (jenting) [inline .github/workflows/lint-test.yaml:42]: This is intended to use `v1.0.0-alpha.3` version to test backward-compatible with helm2.
So, the version should not be bumped. [1]

File: .github/workflows/lint-test.yaml [hunk 2]
```diff
@@ -39,16 +39,16 @@ jobs:
...
 
       - name: Create kind cluster
-        uses: helm/kind-action@v1.0.0-rc.1
+        uses: helm/kind-action@v1.0.0
         # Only build a kind cluster if there are chart changes to test.
         if: steps.lint.outputs.changed == 'true'
 
       - name: Run chart-testing (install)
-        uses: helm/chart-testing-action@v1.0.0-alpha.3 # fixed on v1.0.0-alpha.3 for helm2
+        uses: helm/chart-testing-action@v1.0.0
```
Comment (jenting) [inline .github/workflows/lint-test.yaml:52]: [1]

File: .github/workflows/lint-test.yaml (hunk 1 above)
Comment (cpanato) [reply .github/workflows/lint-test.yaml:42]: humm, ok, thanks!

Comment (cpanato) [reply .github/workflows/lint-test.yaml]: done @jenting

**Pull request merged**